nuget/System.Time.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>System.Time</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/dotnet/corefxlab</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Date, TimeOfDay</description>
+    <releaseNotes>Initial package</releaseNotes>
+    <copyright> Microsoft Corporation.  All rights reserved.</copyright>
+    <tags>Date and Time helper classes</tags>
+  </metadata>
+  <files>
+    <file src="..\bin\Windows_NT.AnyCPU.Release\System.Time\System.Time.dll" target="lib/portable-net45+win8+wpa81+aspnetcore50"/>
+  </files>
+</package>
```

Comment:
I think you need to add the dependencies here. See how it's done in the System.Text.Formatting (which BTW is just an example)